js/views/storyeditview/storyeditview.js

```diff
@@ -261,15 +261,15 @@ var StoryEditView = Marionette.CompositeView.extend(
   },
 
   addPassageEndGameGroupSuccessNumberResult: function(name, left, top) {
-    this.addPassage(name, left, top, 'addPassageEndGameGroupSuccessNumberResult');
+    this.addPassage(name, left, top, 'passageEndGameGroupSuccessNumberResult');

```

Comment:
These strings here are another place we can do `PassageEndGameGroupSuccessNumberResult.prototype.defaults.type` instead of `'passageEndGameGroupSuccessNumberResult'`.

As an aside, that was a really long class name to type out and a lot to read. Not necessarily something to worry about with this PR, but do you think the name could be shortened a bit and still remain descriptive of what it does? like, would `passageEndGameGroup` capture it?